script/deploy/DeployTwabRewards.s.sol

```diff
@@ -0,0 +1,23 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity 0.8.19;
+
+import { ERC20 } from "openzeppelin/token/ERC20/ERC20.sol";
+
+import { TwabController } from "pt-v5-twab-controller/TwabController.sol";
+import { TwabRewards } from "pt-v5-twab-rewards/TwabRewards.sol";
+
+import { Helpers } from "../helpers/Helpers.sol";
+
+contract DeployTwabRewards is Helpers {
+
+  function _deployTwabRewards() internal {
+    TwabController _twabController = _getTwabController();
+    new TwabRewards(_twabController);
```

Comment:
Could be directly passed to the function.
`new TwabRewards(_getTwabController());`

Reply:
There's no need to gas optimize here xD

But if you'd like to change it, I'm indifferent :)